lms/static/js/dashboard/legacy.js

```diff
@@ -181,5 +210,84 @@
              $('#unenroll_course_number').text($(event.target).data('course-number'));
              $('#unenroll_course_name').text($(event.target).data('course-name'));
          });
+
+
+         function set_dialog_attributes(can_unenroll, is_paid_course, is_course_blocked, cert_name_long, course_number,
```

Comment:
I don't really like this function. You are trying to customize the wording display according to the information passed in. Why can't we use mako template in server to achieve this?